src/util.cpp

```diff
@@ -47,7 +47,7 @@ bool ptr_eq(const void *a, const void *b) {
 // Ported from std/mem.zig.
 bool SplitIterator_isSplitByte(SplitIterator *self, uint8_t byte) {
     for (size_t i = 0; i < self->split_bytes.len; i += 1) {
-        if (byte == self->split_bytes.ptr[i]) {
+        if (byte == self->split_bytes.ptr[i] || byte == 0) {
```

Comment:
What was this for? I'm reverting this change - this is a bandage, but let's fix the actual bug.

Reply:
see beginning of thread

> the `chf->path` is the same path but 1 character longer (extra zero)

Reply:
This likely represents a bug in os.cpp having to do with file paths that needs to be fixed. I'll see if I can reproduce it.

Reply:
Here was the real fix: https://github.com/ziglang/zig/commit/d770333827549e5dcbb727c7f434eaac9dd433d2#diff-bf5785092d19ff73388acb473305d70aR1459